pages/pipelines/links_and_images_in_log_output.md.erb

```diff
@@ -84,12 +84,16 @@ Be careful to ensure the part of the URL after `artifact://` exactly matches the
 
 The image artifact does not have to be uploaded at the time it’s written to the build log. If the artifact has not been uploaded you'll see a loading placeholder, and as soon as it's ready the image will automatically appear.
 
-### Library Support
+<section class="Docs__note">
+  <p>If you are using private artifacts, your images need to be base64-encoded to allow Buildkite to access and inline them.</p>
```

Comment:
```suggestion
  <p>If you are using private artifacts, your images need to be <a href="#images-base64-encoded-images">base64-encoded</a> so that Buildkite can access and inline them.</p>
```